### PR TITLE
Re-order Build Tasks to Publish to the Ballerina Central

### PR DESCRIPTION
**Pull request by Bhashinee**
## Purpose
$subject.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -240,6 +240,7 @@ release {
     failOnPublishNeeded = false
 
     buildTasks = ['build']
+    failOnSnapshotDependencies = true
```
Comment (ayeshLK) [inline build.gradle:243]:
```suggestion
    failOnPublishNeeded = false
    failOnSnapshotDependencies = true
    
    buildTasks = ['build']
```

* I think this way it is much more readable.

**Review by aashikam — APPROVED** [review]
LGTM

**Review by ThisaruGuruge — APPROVED** [review]
LGTM, but agree with Ayesh's suggestion.

**Comment by aashikam** [comment]
> LGTM

Also, shall put the `else` clause for `if (ballerinaCentralAccessToken != null) {` in `ballerinaPublish`? @Bhashinee

**Review by aashikam — APPROVED** [review]

**Pull request merged**